common-data/rh.json

```diff
@@ -0,0 +1,8 @@
+{
+    "DHCR_EMAIL_SUBJECT": "Request for Rental History",
+    "DHCR_EMAIL_BODY_PART_1": "I, ",
+    "DHCR_EMAIL_BODY_PART_2": ", am currently living at ",
+    "DHCR_EMAIL_BODY_PART_3": " in apartment ",
+    "DHCR_EMAIL_BODY_PART_4": " and would like to request the complete rent history for this apartment back to the year 1984.",
+    "DHCR_EMAIL_SIGNATURE": "Thank you,"
+}
```

Comment:
For some reason I thought we agreed on a pseudo-template based approach, e.g. a single string called `DHCR_EMAIL_BODY_TEMPLATE` whose value might be something like:

```
I, {{FULL_NAME}}, am currently living at {{ADDRESS}} in apartment {{APARTMENT}}
and would like to request the complete rent history for this apartment
back to the year 1984.
```

and then in JS/python we would essentially interpolate the strings using some ad-hoc string replacement code or perhaps a fancy [substitution regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter).  This might be a bit easier to use and read but it's up to you.

Reply:
we totally did and I forgot! on it, now